sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTransliterateOptions.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System.Collections.Generic;
+
+namespace Azure.AI.Translation.Text
+{
+    /// <summary> Client options for TextTranslationClient.Transliterate </summary>
+    public partial class TextTranslationTransliterateOptions
+    {
+        /// <summary>
+        /// Specifies the language of the text to convert from one script to another.
+        /// Possible languages are listed in the transliteration scope obtained by querying the service
```

Comment:
A consideration for a future PR: Should the `scope` of `GetLanguages` be an extensible enum rather than a string?

Reply:
I have added this question to our list for the architecture review.